src/btree/bt_io.c

```diff
@@ -183,13 +182,15 @@ __wt_bt_write(WT_SESSION_IMPL *session, WT_ITEM *buf,
 	WT_KEYED_ENCRYPTOR *kencryptor;
 	WT_PAGE_HEADER *dsk;
 	size_t dst_len, len, result_len, size, src_len;
+	uint64_t start, stop;
```

Comment:
How did you choose these as a use case to switch over to `rdtsc`?

Reply:
As mentioned in the initial comment in the PR. I looked for ones that are intra-function timings and that I had not already done in WT-3725. I think I'll make this ticket changes complete on their own and remove the changes from the other PR.